version.props

```diff
@@ -1,6 +1,6 @@
 <Project>
     <PropertyGroup>
-        <VersionPrefix>1.0.0</VersionPrefix>
-        <VersionSuffix>msbuild4</VersionSuffix>
+        <VersionPrefix>1.1.0</VersionPrefix>
+        <VersionSuffix>msbuild3</VersionSuffix>
```

Comment:
...I assume you mean to get this in for RC.3...does this mean we also need to update dotnet-migrate to know when to use 1.0 vs 1.1?